packages/nodejs/src/config/configmap.ts

```diff
@@ -17,6 +17,10 @@ export const ENV_TO_KEY_MAPPING: { [key: string]: string } = {
   APPSIGNAL_IGNORE_ACTIONS: "ignoreActions",
   APPSIGNAL_IGNORE_ERRORS: "ignoreErrors",
   APPSIGNAL_IGNORE_NAMESPACES: "ignoreNamespaces",
+  APPSIGNAL_INSTRUMENT_HTTP: "instrumentHttp",
+  APPSIGNAL_INSTRUMENT_HTTPS: "instrumentHttps",
```

Comment:
```suggestion
```

Since the HTTP and HTTPS settings have been merged into `instrumentHttp`, the `instrumentHttps` entry can be removed from this map.